src/components/Signup/Signup.jsx

```diff
@@ -1,129 +1,131 @@
 import React from 'react';
-import { TextArea, Form, Modal, Select } from 'semantic-ui-react';
-
-import TabBar from '../TabBar/TabBar';
-import SignUpButton from '../Shared/SignUpButton'
+import { TextArea, Form, Modal, Select, Popup, Icon, Segment } from 'semantic-ui-react';
 import { states } from '../../utils/states';
 import { months } from '../../utils/month';
 import { dates } from '../../utils/dates';
 import { years } from '../../utils/year';
 
+import TabBar from '../TabBar/TabBar';
+import SignUpButton from '../Shared/SignUpButton';
+
 const ModalSignUp = (
-  { 
-    first_name, 
-    last_name,
-    username, 
-    retype_password,
-    password, 
-    email,
-    month,
-    date,
-    year,
-    city,
-    state,
-    zipcode,
-    about,
-    submittedName,
-    submittedEmail, 
-    handleChange, 
-    handleSubmit,
-    modalOpen,
-    closeModal,
-    closeConfig,
-    newClose,
-    validations,
-  }
+	{  
+		first_name, 
+		last_name,
+		username,
+		retype_password,
+		password, 
+		email,
+		month,
+		date,
+		year,
+		city,
+		state,
+		zipcode,
+		about,
+		submittedName,
+		submittedEmail, 
+		handleChange, 
+		handleSubmit,
+		upModalOpen,
+		upClose,
+		upCloseConfig,
+		newClose,
+		signBothClose,
+		validations,
+	}
```

Comment:
I'm starting to realize that there's huge spacing between each line. Was this intended?